packages/deployment/app/cumulus_api.template.yml

```diff
@@ -197,7 +197,13 @@ Resources:
             Fn::Join: ["", [ "https://", {"Ref": "downloadRestApi"}, ".execute-api.", {"Fn::Sub": "${AWS::Region}"}, ".amazonaws.com/{{../parent.apiStage}}/redirect"]]
         {{/if}}
       {{/if}}
+  {{# if (lookup ../parent.communityConfiguration @key)}}
```

Comment:
I vote for the handling the auth logic in the lambda code rather than in the template.